src/crypto/crypto_keys.cc

```diff
@@ -489,8 +489,13 @@ std::shared_ptr<KeyObjectData> ImportJWKSecretKey(
 Maybe<bool> ExportJWKAsymmetricKey(
     Environment* env,
     std::shared_ptr<KeyObjectData> key,
-    Local<Object> target) {
+    Local<Object> target,
+    bool handleRsaPss) {
```

Comment:
I'd prefer a named flag than a bool here. Something like

```
enum class ExportJWKFlags {
  NONE,
  FULL,
}
```

Reply:
Could you suggest the full changeset?

Reply:
When I'm back on my laptop and not my phone yes :-)

Reply:
Thank you